aptos-move/aptos-gas/src/transaction.rs

```diff
@@ -20,7 +20,7 @@ crate::params::define_gas_parameters!(
         [
             min_transaction_gas_units: InternalGas,
             "min_transaction_gas_units",
-            600
+            1_500_000
```

Comment:
What is this for?

Reply:
This is the minimum gas fee any transaction would need to pay, even if they fail early in the pipeline.